docker-compose/monitor/README.md

```diff
@@ -83,6 +85,40 @@ Then navigate to the Monitor tab at http://localhost:16686/monitor to view the R
 
 ![My Service RED Metrics](images/my_service_metrics.png)
 
+## Migrating to Span Metrics Connector 
+
+### Background
+
+A new [Connector](https://pkg.go.dev/go.opentelemetry.io/collector/connector#section-readme) component was introduced
```

Comment:
```suggestion
A new [Connector](https://pkg.go.dev/go.opentelemetry.io/collector/connector#section-readme) API was introduced
```

Reply:
Renamed in: [d5752b2](https://github.com/jaegertracing/jaeger/pull/4452/commits/d5752b267c9ba92ddf60657a8973e7ff8ce66801)